src/usb_cam.cpp

```diff
@@ -1124,19 +1125,20 @@ void UsbCam::grab_image()
   if (-1 == r)
   {
     if (EINTR == errno)
-      return;
+      return true;
 
     errno_exit("select");
```

Comment:
shouldn't we `return false` here? https://man7.org/linux/man-pages/man2/select.2.html#RETURN_VALUE

Reply:
no, because based on the original logic when it returns empty then the code will resume to the caller. This is equivalent to `return true` here, if we `return false` then the node is shut down which we don't want to.

Reply:
I mean the `errno_exit()` that calls `exit()` function, from my understanding this is what we are changing. Right?

```suggestion
   return false;
```

Reply:
Yeah I guess, I just made the fixup. I didn't make it at first because we haven't seen this error encountered and this PR was just trying to fix the other error exit process that caused the select timeout error. But I guess I can make this PR more general.